polynote-server/src/main/scala/polynote/server/repository/ipynb/ast.scala

```diff
@@ -86,7 +86,8 @@ object JupyterOutput {
       args.get("rel") match {
         case Some(name) if mime == "text/plain" => Stream(name, content.linesWithSeparators.toList)
         case Some("output") => DisplayData(Map(mime -> Json.arr(content.linesWithSeparators.toSeq.map(_.asJson): _*)), None)
-        case None => ExecuteResult(execId, Map(mime -> Json.arr(content.linesWithSeparators.toSeq.map(_.asJson): _*)), None)
+        case Some(_) | None => // do we want to do anything with the rel value?
```

Comment:
I wonder if ipynb would allow the whole `contentType` string (with args) in the mime type – it's a legal MIME type, but ipynb seems sensitive.

Reply:
Ah nice catch, the colors with this setup don't show up if I open the notebook in a fresh session, nice catch. 

I can try to test it out by launching a jupyter session and trying to load the notebook (unless there's a better way!).

Reply:
bad news - it looks like if data mimetype doesn't exactly match then it won't render in jupyter 😢 

I have two ideas on how to solve this (open to more of course!):

1. Multiple outputs: 
```
      "outputs" : [
        {
          "execution_count" : 1,
          "data" : {
            "text/plain" : [
              "resCell1: List[Int] = List(1, 2, 3)"
            ],  <--- jupyter will read this
            "text/scala" : [
              "resCell1: List[Int] = List(1, 2, 3)"
            ]   <--- we would have to read this and ignore the text/plain
          },
          "output_type" : "execute_result"
        }
      ]
    },
```
2. Use metadata
```
      "outputs" : [
        {
          "execution_count" : 1,
          "data" : {
            "text/plain" : [
              "resCell1: List[Int] = List(1, 2, 3)"
            ]
          "output_type" : "execute_result"
        },
        "metadata": {
          "lang": "scala"
        },
      ]
    },
```

Glad to know what you think!